### PR TITLE
Added update payment endpoint

### PR DESCRIPTION
**Pull request by bairixie**
Made the following changes:
-Added update payment endpoint in 'routes.py'
-Added test case for payment endpoint

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -27,69 +27,61 @@
...
 
 
+
+
 ######################################################################
-# GET INDEX
+# UPDATE AN EXISTING Payment
 ######################################################################
-@app.route("/")
-def index():
```
Comment (nedjulius) [inline service/routes.py:34]: it seems that you removed some code that should belong in master branch during your merge attempt

File: tests/test_routes.py [hunk 2]
```diff
@@ -61,24 +47,4 @@ def test_index(self):
         self.assertEqual(data["version"], "1.0")
         self.assertEqual(len(methods), 5)
 
-        # check if root path has definitions for all methods
-        def is_path_and_method_in_list(path, method):
```
Comment (nedjulius) [inline tests/test_routes.py:65]: same here

File: tests/test_routes.py [hunk 3]
```diff
@@ -9,48 +9,34 @@
...
 DATABASE_URI = os.getenv(
     "DATABASE_URI", "postgresql+psycopg://postgres:postgres@localhost:5432/postgres"
 )
-
-
-######################################################################
-#  T E S T   C A S E S
-######################################################################
-# pylint: disable=too-many-public-methods
-class TestPaymentsService(TestCase):
```
Comment (nedjulius) [inline tests/test_routes.py:21]: same here

**Review by nedjulius — CHANGES_REQUESTED** [review]
your additions look alright, but you removed some code from the master branch. could you create a new PR or just revert changes to before the branch merge?

File: tests/test_routes.py [hunk 4]
```diff
@@ -9,48 +9,34 @@
...
+        # update the payment
+        new_payment = response.get_json()
+        logging.debug(new_payment)
+        new_payment["name"] = test_payment.name
+        response = self.client.put(f"{BASE_URL}/{new_payment['id']}", json=new_payment)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        updated_payment = response.get_json()
+        self.assertEqual(updated_payment["name"], test_payment.name)
+        
         """It should call the home page and receive information about existing methods"""
```
Comment (nedjulius) [inline tests/test_routes.py:40]: these should be separate test cases. it seems you removed part of the `index` test case and then merged the rest of it with your test case, even though they test different thing

File: service/routes.py (hunk 1 above)
Comment (bairixie) [reply service/routes.py:34]: fixed

**Pull request closed without merging**